src/preface.md

```diff
@@ -1 +1,70 @@
+<p style="text-align:center;"><img src="./assets/esp-logo-black.svg" width="50%"></p>
+
 # Preface
+
+Welcome to a comprehensive guide to using the `esp-rs` ecosystem, created for embedded Rust development on Espressif products. This book serves as a practical resource for developers wishing to utilise the capabilities of Rust in `no_std` environments when working with Espressif hardware platforms.
+
+## Who This Book Is For
+
+This book is intended for people with some experience in Rust and assumes a basic knowledge of embedded development and electronics. The expected target user of our product and reader of this book has a basic understanding of concepts such as cross-compilation, common digital interfaces (UART, SPI, I2C), memory-mapped peripherals and interrupts. For those without prior experience, we recommend first reading the [Scope of This Book][prerequisites] and [Resources][resources] sections to get up to speed.
```

Comment:
```suggestion
This book is intended for people with some experience in Rust and assumes basic knowledge of embedded development and electronics. The expected target user of our product and reader of this book has a basic understanding of concepts such as cross-compilation, common digital interfaces (UART, SPI, I2C), memory-mapped peripherals and interrupts. For those without prior experience, we recommend first reading the [Scope of This Book][prerequisites] and [Resources][resources] sections to get up to speed.
```

Also, what does "product" refer to here? I think this should be reworded as it's pretty ambiguous, IMO.